utils/build-script

```diff
@@ -701,6 +701,11 @@ class BuildScriptInvocation(object):
                 "--host-lipo", toolchain.lipo,
             ]
 
+            # Isolate build from the system; Darwin toolchains build against SDKs.
+            args.extra_cmake_options.append(
+                '-DCMAKE_IGNORE_PATH=/usr/lib;/usr/local/lib;/lib'
```

Comment:
Can you add to the comment here what type of problems result from us not doing this?

Reply:
Or you know what. I think this is pretty common sense. Nm.